cmake/Utils.cmake

```diff
@@ -69,19 +80,23 @@ function(format_gencode_flags flags out)
   endif()
   # Generate SASS
   foreach(ver ${flags})
-    set(${out} "${${out}}-gencode arch=compute_${ver},code=sm_${ver};")
+    set(${out} "${${out}}--generate-code=arch=compute_${ver},code=sm_${ver};")
   endforeach()
   # Generate PTX for last architecture
   list(GET flags -1 ver)
-  set(${out} "${${out}}-gencode arch=compute_${ver},code=compute_${ver};")
-  
+  set(${out} "${${out}}--generate-code=arch=compute_${ver},code=compute_${ver};")
+
   set(${out} "${${out}}" PARENT_SCOPE)
 endfunction(format_gencode_flags flags)
 
 # Assembles the R-package files in build_dir;
 # if necessary, installs the main R package dependencies;
 # runs R CMD INSTALL.
 function(setup_rpackage_install_target rlib_target build_dir)
+  # backup cmake_install.cmake
```

Comment:
What is this for?

Reply:
@hcho3 The next line removes whole R-package build directory, resulting in a missing `cmake_install.cmake` file.  That will cause installation failure.